src/table/components/TableWrapper.tsx

```diff
@@ -38,15 +39,15 @@ export default function TableWrapper(props) {
   const focusedCellCoord = useContextSelector(TableContext, (value) => value.focusedCellCoord);
```

Comment:
There is a conflict on `value.focusedCellCoord` that I've faced the same in `TableTotals` but couldn't solve it. Have you had any chance to fix it ?

Reply:
Sure. Which PR should I look at?

Reply:
No I mean in your PR, the current PR. line 38 and 39 have type conflict, or it looks like it has when I opened this file in your branch.

Reply:
no, i do not see any. if it has type conflicts, the `ci/circleci: build` should fail.

Reply:
Donya the error was solved by adding a type to the context https://github.com/qlik-oss/sn-table/pull/650/files#diff-dfc3f242cfc56f21ea352b2b1bc30eacad5c9fef36a416af3a6da11f20c393b3R8

Reply:
Ah Ok, I saw it now. Thanks!